packages/salesforcedx-sobjects-faux-generator/test/integration/fauxGenerate.test.ts

```diff
@@ -0,0 +1,200 @@
+/*
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { SFDX_PROJECT_FILE } from '@salesforce/salesforcedx-utils-vscode/out/src';
+import { LocalCommandExecution } from '@salesforce/salesforcedx-utils-vscode/out/src/cli';
+import rimraf = require('rimraf');
+import { expect } from 'chai';
+import { EventEmitter } from 'events';
+import { renameSync } from 'fs';
+import * as path from 'path';
+import { SObjectCategory } from '../../src/describe';
+import {
+  FAILURE_CODE,
+  FauxClassGenerator,
+  SUCCESS_CODE
+} from '../../src/generator/fauxClassGenerator';
+import { nls } from '../../src/messages';
+import * as util from './integrationTestUtil';
+
+// The CustomObjects are all identical in terms of fields, just different ones to test batch
+// and multiple objects for testing describeGlobal
+const PROJECT_NAME = `project_${new Date().getTime()}`;
+const CUSTOM_OBJECT_NAME = 'MyCustomObject__c';
+const CUSTOM_OBJECT2_NAME = 'MyCustomObject2__c';
+const CUSTOM_OBJECT3_NAME = 'MyCustomObject3__c';
+const CUSTOM_FIELD_FULLNAME = '.MyCustomField__c';
+const SIMPLE_OBJECT_SOURCE_FOLDER = 'simpleObjectAndField';
+
+// tslint:disable:no-unused-expression
+describe('Generate faux classes for SObjects', function() {
```

Comment:
Why function()? I thought we decided to use the `() = > {` syntax per other tests files?

Reply:
just a mistake in copy/paste from a different test... oops

Reply:
@GaryFNC - So this has not been fixed yet. You are still using the old syntax.